pkg/forwarder/transaction.go

```diff
@@ -123,6 +123,16 @@ func initTransactionExpvars() {
 	transactionsErrorsByType.Set("SentRequestErrors", &transactionsSentRequestErrors)
 }
 
+/// Define the priority of a transaction
+/// Transactions with priority `TransactionPriorityNormal` are dropped from the retry queue
+/// before dropping transactions with priority `TransactionPriorityHigh`.
+type TransactionPriority int
+
+const (
+	TransactionPriorityNormal TransactionPriority = 0
+	TransactionPriorityHigh   TransactionPriority = 1
+)
+
```

Comment:
As we may need to introduce a `medium` priority or anything else between `normal` and `high` in an uncertain future, maybe we can set `TransactionPriorityHigh   TransactionPriority = 10` to have some room for intermediate priority.

Reply:
What prevent us from updating `TransactionPriorityHigh` to 10 later?

Reply:
Nothing, it is just in case it ends up reused and/or hardcoded in other places.